napps/kytos/of_l2ls/main.py

```diff
@@ -51,6 +52,9 @@ def handle_packet_in(self, event):
 
         switch.update_mac_table(ethernet.source, in_port)
 
+        if packet_in.reason != PacketInReason.OFPR_NO_MATCH:
+            return
```

Comment:
If a napp wants to listen to packet-ins and do whatever they want, there's nothing stopping them.
Certainly something must be done with the explicitly forwarded packages, but not flood or flow_mod, right? that' s what this line is avoiding.